help/grandorgue.xml

```diff
@@ -5652,6 +5652,52 @@ role="bold">Group</emphasis> attribute are grouped in the same submenu.
 	</sect3>
       </sect2>
     </sect1>
+    <sect1>
+      <title>Enclosures</title>
+      <indexterm><primary>Organ settings.Enclosures</primary></indexterm>
+      <figure>
+	<title>Organ Settings dialog, Enclosures tab</title>
+	<mediaobject>
+	  <imageobject>
+	    <imagedata scalefit="1" width="100%" fileref="images/organ_settings_enclosures.png"/>
+	  </imageobject>
+	</mediaobject>
+      </figure>
+      <para>
+	In its left box, this dialog displays all enclosures: both added 
+	internally by GrandOrgueand ones found in the definition file. Now only
+	the enclosures added internally are allowed to be changed.
+      </para>
+      <simpara>
+	Select the element you wish to customize. The right controls are
+	populated with values picked from the organ definition file or the
+	current preset number if the latter was saved.
+      </simpara>
+      <simpara>
+	It is possible to select multiple elements at once, to e.g. update one
+	setting of multiple elements to the same value in a single operation.
+      </simpara>
+      <sect2>
+	<title>Affected windchests</title>
+	<para>
+	  The list of windchest the enclosure affects.
+	</para>
+	<para>
+	  The same enclosure appears under all the the windchests in tree.
```

Comment:
I don't exactly understand what this line means right now though...

Reply:
Removed